packages/common/src/utils/validationUtils.test.ts

```diff
@@ -1381,20 +1382,20 @@ describe('autoFillElement', () => {
 describe('Validate expressions with boolean values set to null or undefined should not be showed', () => {
   const element = getBooleanElement()
 
-  it('should return false if selected is undefined for negative expression', () => {
+  it('should return true if selected is undefined for negative expression', () => {
     ;(element.value as ValueBoolean).selected = undefined
     const result = parseExpression('!$dodsdatumSakert', element, CertificateDataValidationType.SHOW_VALIDATION)
-    expect(result).toBe(false)
+    expect(result).toBe(true)
   })
```

Comment:
This could potentially introduce a breaking change, but the previous evaluation doesn't make sense to me `!undefined` or `!null` should resolve to `true`